src/components/ChatEntry.js

```diff
@@ -3,20 +3,31 @@ import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  let className = 'local';
+
+  if (props.id % 2 === 0) {
+    className = 'chat-entry local';
+  } else {
+    className = 'chat-entry remote';
+  }
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={className}>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
+        <p> {props.body} </p>
+        <p className="entry-time">{props.timeStamp}</p>
```

Comment:
There is a pre-provided Timestamp component that should have been used here to properly render the timestamp.